readme.txt

```diff
@@ -129,6 +129,7 @@ Currently, the following add-ons are available for Event Tickets:
 * Tweak - Update status manage to accept provider names or abbreviations [120862]
 * Tweak - In the Ticket Block add link to EDD Orders Page [121440]
 * Tweak - Change "Attendee Registration" to "Attendee Information" in several locations [126038]
+* Tweak - Supported post types for tickets by removing product and download from saved options to prevent recursion errors [126749]
```

Comment:
```suggestion
* Tweak - Exclude WooCommerce Product and EDD Downloads as supported post types for tickets to prevent recursion errors [126749]
```

Reply:
Is this fix only on save or does it no longer show them in the form either now?

Reply:
They were not showing in the form. They upgraded and the values were still saved from an older version.